Tests/Common/SimulatedBasicsStation.cs

```diff
@@ -147,6 +144,15 @@ public async Task StopAndValidateAsync(CancellationToken cancellationToken = def
             }
         }
 
+        private static ClientWebSocket CreateClientWebSocket()
+        {
+            var result = new ClientWebSocket();
+#pragma warning disable CA5359 // Do Not Disable Certificate Validation
```

Comment:
Let's add a comment saying that we are making use of self-signed certificates on the LNS that we are targeting for Load Tests and therefore we need to either disable the validation or trust the cert (we chose the first)

Reply:
98c2a895d0ba721431d1f824aeb5d2c43f421b92